specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/examples/FileShareAclsPatch.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res3376",
+    "accountName": "sto328",
+    "shareName": "share6185",
+    "api-version": "2021-04-01",
+    "monitor": "true",
+    "fileShare": {
+      "properties": {
+        "signedIdentifiers": [
+          {
+            "id": "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTI",
+            "accessPolicy": {
+              "start": "2021-04-01T08:49:37.0000000Z",
+              "expiry": "2021-05-01T08:49:37.0000000Z",
+              "permission": "rwd"
+            }
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/res3376/providers/Microsoft.Storage/storageAccounts/sto328/fileServices/default/shares/share6185",
+        "name": "share6185",
+        "type": "Microsoft.Storage/storageAccounts/fileServices/shares",
+        "properties": {
+          "signedIdentifiers": [
+            {
+              "id": "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTI",
+              "accessPolicy": {
+                "start": "2021-04-01T08:49:37.0000000Z",
+                "expiry": "2021-05-01T08:49:37.0000000Z",
```

Comment:
In Update Share with "signedIdentifiers", in my test, start/expire are missing in the responds. （Then Get share also miss start/expire. Seems start/expire are not saved to server）

```
PATCH https://management.azure.com/subscriptions/***/resourceGroups/***/providers/Microsoft.Storage/storageAccounts/***/fileServices/default/shares/share2939?api-version=2021-04-01 HTTP/1.1
x-ms-client-request-id: 2d9e2be3-022c-4d1a-bdbf-28d6b9a00708
Accept-Language: en-US
Authorization: Bearer 
User-Agent: FxVersion/4.6.29916.01 OSName/Windows OSVersion/Microsoft.Windows.10.0.19042. Microsoft.Azure.Management.Storage.StorageManagementClient/21.2.0.0
Content-Type: application/json; charset=utf-8
Content-Length: 502
Host: management.azure.com

{
  "properties": {
    "signedIdentifiers": [
      {
        "id": "testSig1",
        "accessPolicy": {
          "start": "2021-05-07T10:00:03.1537718Z",
          "expiry": "2021-05-07T12:00:03.1537763Z",
          "permission": "rw"
        }
      },
      {
        "id": "testSig2",
        "accessPolicy": {
          "start": "2021-05-07T11:00:03.1537774Z",
          "expiry": "2021-05-07T14:00:03.1537778Z",
          "permission": "rwdl"
        }
      }
    ]
  }
}
HTTP/1.1 200 OK
Cache-Control: no-cache
Pragma: no-cache
Content-Length: 399
Content-Type: application/json
Expires: -1
ETag: "0x8D9113EE28591C4"
x-ms-request-id: ecd8f652-6c62-4c39-8248-071eb534efcc
Strict-Transport-Security: max-age=31536000; includeSubDomains
Server: Microsoft-Azure-Storage-Resource-Provider/1.0,Microsoft-HTTPAPI/2.0 Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-writes: 1196
x-ms-correlation-request-id: 1cafeb2f-4ef2-4438-b5a5-f607261cc1c8
x-ms-routing-request-id: SOUTHEASTASIA:20210507T100003Z:1cafeb2f-4ef2-4438-b5a5-f607261cc1c8
X-Content-Type-Options: nosniff
Date: Fri, 07 May 2021 10:00:02 GMT

{"id":"/subscriptions/***/resourceGroups/***/providers/Microsoft.Storage/storageAccounts/***/fileServices/default/shares/***","name":"share2939","type":"Microsoft.Storage/storageAccounts/fileServices/shares","properties":{"signedIdentifiers":[{"id":"testSig1","accessPolicy":{"permission":"rw"}},{"id":"testSig2","accessPolicy":{"permission":"rwdl"}}]}}
```